syntax = "proto3";

package suggestions;

service Suggestions {
    rpc GetBookSuggestions (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
    string user_id = 1;
    int32 num_suggestions = 2;
}

message SuggestionsResponse {
    repeated string books = 1;
}
